multi-lock/main.ts

```diff
@@ -0,0 +1,130 @@
+// deno-lint-ignore-file
+import { parse } from "https://deno.land/std@0.170.0/flags/mod.ts";
+
+import { ApiVersion, PhylumApi } from "phylum";
+
+const USAGE = `
+phylum multi-lock [OPTIONS] <LOCKFILE>...
+
+Arguments:
+  <LOCKFILE>...  The package lockfile(s) to submit.
+
+Options:
+  -l, --label <label>           Specify a label to use for analysis
+  -p, --project <project_name>  Specify a project to use for analysis
+  -g, --group <group_name>      Specify a group to use for analysis
+  -h, --help                    Print help information
+`.trim();
+
+const args = parse(
+  Deno.args,
+  {
+    unknown: (_a, k?) => {
+      if (k) {
+        console.error(`ERROR: Unexpected argument: ${k}`);
+        console.log(USAGE);
+        Deno.exit(-1);
+      }
+    },
+    boolean: ["help"],
+    string: ["_", "label", "project", "group"],
+    alias: {
+      "l": "label",
+      "p": "project",
+      "g": "group",
+      "h": "help",
+    },
+  } as const,
+);
+
+if(args.help) {
+  console.log(USAGE);
+  Deno.exit(0);
+}
+
+if (!args._.length) {
+  console.error("ERROR: No lockfile provided");
+  console.log(USAGE);
+  Deno.exit(-1);
+}
+
+type Package = { name: string; version: string; type: string };
+const packages = await Promise.all(
+  args._.map((lockfile) =>
+    PhylumApi.parseLockfile(lockfile as string).then((l: any) =>
+      l.packages.map((p: any) => ({ ...p, type: l.package_type })) as Package[]
```

Comment:
I'd like it better if the `any` instances here were strong types, but I suppose that's contingent on having type definitions in the global shim first. Just in case, I'd create a ticket to track this so we can come to it later.

Reply:
I completely agree. And all extensions will have to do something like this until we add type definitions to the global shim.

Instead of creating an issue to track this specific block of code, I've created #15 because we don't currently enforce any formatting or linting on this repository.

And this triggers [`no-explicit-any`](https://lint.deno.land/?q=any#no-explicit-any) in `deno lint`. So that will be our reminder to come back and clean this up.